posts/templates/posts/_post_create.html

```diff
@@ -13,16 +13,17 @@
     <br/>
 
     <div class="row">
-        {% if user.circles %}
-            <div class="form-group col-xs-12 col-md-4">
+        <div class="form-group">
+            <div class="col-xs-12 col-md-4">
                 <select class="form-control" name="circle">
-                    <option disabled selected="selected">Select a circle</option>
-                    {% for circle in user.circles|dictsort:'name' %}
-                        <option value="{{ circle.pk }}">{{ circle.name }}</option>
+                    <option value="-2"{% if post.circles_id == -2 %} selected{% endif %}>Only visible to me</option>
```

Comment:
I would also be more specific. `Me` is to short. Facebook says `Only me` but why not `Only visible to me`. :wink: